server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/PartitionFilesSorter.java

```diff
@@ -97,10 +97,10 @@ public int getSortingCount() {
     return shuffleSortTaskDeque.size();
   }
 
-  public FileInfo openStream(String shuffleKey, String fileName, FileWriter fileWriter,
+  public FileInfo openStream(String shuffleKey, String fileName, LocalFileMeta fileMeta,
```

Comment:
Fileinfo and LocalFileMeta are sort of similar. Maybe they can be merged.

Reply:
+1, I think FileInfo is qualified to replace LocalFileMeta

Reply:
Yea, nice suggestion